client/my-sites/earn/home.tsx

```diff
@@ -501,7 +497,18 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 			align: 'right',
 		},
 		body: translate(
-			'Accept credit card payments today for just about anything – physical and digital goods, services, donations and tips, or access to your exclusive content. Turn your website into a reliable source of income with payments and ads.'
+			'Accept credit card payments today for just about anything – physical and digital goods, services, donations and tips, or access to your exclusive content. {{a}}Watch our tutorial videos to get started{{/a}}.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **11** times:
`translate( 'Accept credit and debit card payments today for just about anything – physical and digital goods, services, donations and tips, or access to your exclusive content. {{link}}Learn how to get started{{/link}}.' )` *ES Score: 7*